staging/src/k8s.io/pod-security-admission/policy/checks.go

```diff
@@ -137,12 +145,16 @@ func AggregateCheckResults(results []CheckResult) AggregateCheckResult {
 				reasons = append(reasons, result.ForbiddenReason)
 			}
 			details = append(details, result.ForbiddenDetail)
+			if result.ErrList != nil {
```

Comment:
If you only append when ErrList in non-nil, it could mean that the `ErrLists` items don't line up with the ForbiddenReasons, and you can't tell which reason the error list corresponds to (see previous comment suggesting a map instead)